src/App.js

```diff
@@ -1,16 +1,154 @@
+  function toggleDisable() {
+    document.getElementById("start-button").disabled = true;
+    document.getElementById("guess").disabled = false;
+    document.getElementById("quit").disabled = false;
+    // document.getElementById('East').disabled= false;
+    // document.getElementById('West').disabled= false;
+    // document.getElementById('North').disabled = false;
+    // document.getElementById('South').disabled = false;
+    // document.getElementById('Return').disabled = false;
+  }
+//-------------Function to set up Start, Quit, Guess, and Navigation buttons, with Divs to allow targeted CSS---------//
+  function MainButtons() {
+    return (
+      <div id="allButtons">
+        <h3 class= "needsBG">Movement Buttons</h3>
+        <div id="nav-buttons">
+        
+          <div id="top">
+            <button id="North" class="nav-buttons" onClick={moveNorth}>
+              North
+            </button>
+          </div>
+    {/* Div for East, West, And Center */}
+          <div id="mid3">
+            <button id="West" class="nav-buttons" onClick={moveWest}>
+              West
+            </button>
+            <button id="Return" class="nav-buttons" onClick={returnCenter}>
+              Start Point
+            </button>
+            <button id="East" class="nav-buttons" onClick={moveEast}>
+              East
+            </button>
+          </div>
 
-  const [center, setCenter] = useState([43.88, -72.7317])
+          <div id="bottom">
+            <button id="South" class="nav-buttons" onClick={moveSouth}>
+              South
+            </button>
+          </div>
+        </div>
 
+        <div id = "score" class= "needsBG">
+        <h4>{`Player Score: ${playerScore}`}</h4>
+        </div>
+{/* Div for Play and Quit Buttons and Dropdown Modal for Guessing county */ }
+        <div id="playButtons">
+          <h3 class= "needsBG">GamePlay Options</h3>
+          <button id="start-button" onClick={startGame}>
+            Start a Game
+          </button>
+          <Modal />
+          <button disabled={false} id="quit">
+            I Give Up!
+          </button>
+        </div>
+      </div>
+    );
+
+    //-------------------------functions to move center and deduct point from score----------------//
+
+    function moveEast() {
+      randLon = randLon + 0.002;
+      setCenter([randLat, randLon]);
+      playerDeductPoint(1);
+    }
+
+    function moveWest() {
+      randLon = randLon - 0.002;
+      setCenter([randLat, randLon]);
+      playerDeductPoint(1);
+    }
+
+    function moveNorth() {
+      randLat = randLat + 0.002;
+      setCenter([randLat, randLon]);
+      playerDeductPoint(1);
+    }
+
+    function moveSouth() {
+      randLat = randLat - 0.002;
+      setCenter([randLat, randLon]);
+      playerDeductPoint(1);
+    }
+//-------Function to return to original random point------//
+    function returnCenter() {
+      setCenter(startCenter);
```

Comment:
You also want to change your randLat and randLon variables through here. When you don't, your NSEW buttons pull the last location those were at when moving again